src/test/java/com/diyasylum/formfiller/i589FillerTest.java

```diff
@@ -29,10 +34,47 @@
     return response.body();
   }
 
+  private PDDocument createForm() throws IOException {
```

Comment:
Does this create a blank PDF and put some stuff on it? One future request will be to include a cover page on the application with instructions on further steps that need to be taken (signatures, other documents to be included, where to mail it, etc). For that we'll need to know how to generate our own PDF pages.

Reply:
Im fairly confident we have a lot of flexibility for adding pages, duplicating existing pages, etc etc. Gonna have to learn a bit more about he drawing apis though.

Im like.... 90 percent sure this method is creating an empty file with form elements. But I can find out pretty quickly